pkg/skaffold/kubernetes/portforward/entry_manager.go

```diff
@@ -41,6 +42,19 @@ var (
 			entry.resource.Name,
 			entry.resource.Address)
 	}
+	portForwardEventV2 = func(entry *portForwardEntry) {
+		// TODO priyawadhwa@, change event API to accept ports of type int
```

Comment:
Is this TODO still relevant?
If yes, we should remove @priyawadhwa from here.

Reply:
whoops, that shouldn't be there. Removed the other as well as the event already accepts `int32`